urlshortener/db.py

```diff
@@ -0,0 +1,44 @@
+import azure.cosmos.cosmos_client as cosmos_client
+import azure.cosmos.exceptions as exceptions
+from azure.cosmos.partition_key import PartitionKey
+from os import environ
+
+
+def setup_db():
+    HOST = environ["COSMOS_HOST"]
+    MASTER_KEY = environ["COSMOS_MASTER_KEY"]
+    DATABASE_ID = "urlshortener"
+    CONTAINER_ID = "links"
+    CONTAINER_ID2 = "users"
```

Comment:
I think calling these vars e.g. `LINKS` and `USERS` would be better. However, do you really need two containers? Have you considered using a field to distinguish entity type?